packages/components/src/components/hds/side-nav/header/icon-button.ts

```diff
@@ -18,11 +18,27 @@ interface HdsSideNavHeaderIconButtonSignature {
 }
 
 export default class HdsSideNavHeaderIconButtonComponent extends Component<HdsSideNavHeaderIconButtonSignature> {
-  /**
-   * @param ariaLabel
-   * @type {string}
-   * @description The value of `aria-label`
-   */
+  constructor(
+    owner: unknown,
+    args: HdsSideNavHeaderIconButtonSignature['Args']
+  ) {
+    super(owner, args);
+
+    deprecate(
+      'The `Hds::SideNav::Header::IconButton` sub-component is now deprecated and will be removed in the next major version of `@hashicorp/design-system-components`. Use `Hds::Button` as one-to-one replacement.',
+      false,
+      {
+        id: 'hds.components.sidenav.header.iconbutton',
+        until: '5.0.0', // TODO: Confirm version
+        url: 'https://helios.hashicorp.design/components/side-nav?tab=version%20history#480', // TODO: Confirm URL
+        for: '@hashicorp/design-system-components',
+        since: {
+          enabled: '4.9.0', // TODO: Confirm version
```

Comment:
Seems save to assume this will land in 4.10 as part of https://github.com/hashicorp/design-system/pull/2330

Reply:
But we can always change it directly in the release branch if we need to. Same with the URL, but you could assume it would be https://helios.hashicorp.design/components/side-nav?tab=version%20history#4100 for now

Reply:
Ok, thank you!